ansible/tasks/initdb.yml

```diff
@@ -2,20 +2,20 @@
 # This task will initialize the immortal DBs in the database account.
 # This step is usually done only once per account.
 
-- name: check if the immortal {{ db_auth }} db with {{ db_provider }} exists?
+- name: check if the immortal {{ db.whisk.auth }} db with {{ db_provider }} exists?
```

Comment:
@cbickel have you checked that the vars are replaced? I can recall having to put ".." around the test to have them replaced by ansible.